spec/internal/app/controllers/posts_controller.rb

```diff
@@ -11,6 +12,12 @@ class PostsController < Matterhorn::Base
 
   add_env :current_user
 
+  paginates_with Matterhorn::Paging::Default
+
+  # used for inclusions
+  allow_collection_params \
```

Comment:
I meant to show you the call to add `allow_collection_params :include` is not needed, because it's now handled during builder_support setup.  

In master currently the test app doesn't have that line anymore: https://github.com/blakechambers/matterhorn/blob/9822ee304d36db97e250bc6b9e346f678bf40120/spec/internal/app/controllers/posts_controller.rb